cli/src/commands/file/show.rs

```diff
@@ -57,6 +58,7 @@ pub(crate) struct FileShowArgs {
         value_hint = clap::ValueHint::FilePath,
         add = ArgValueCompleter::new(complete::all_revision_files),
     )]
+    #[arg(value_name = "FILESETS")]
```

Comment:
nit: combine `arg()`s for consistency?

(applies to file/show.rs, file/untrack.rs, interdiff.rs, log.rs, restore.rs, squash.rs)

Reply:
Oops, done. Thanks.